nautobot/extras/templates/django_ajax_tables/ajax_wrapper.html

```diff
@@ -0,0 +1,48 @@
+<!--
+This is a fork of `ajax_wrapper.html` from `django-ajax-tables` plugin core. It has been updated to store the current query string in session storage so that it may be passed along to persist current page in the log table while a job is running.
+
+Ref: https://github.com/nautobot/nautobot/issues/1289
+-->
+
+<div id="{{ ajax_divname }}"></div>
+
+<script type='text/javascript'>
+
+    function _replace_links_{{ ajax_divname }}(url, elements) {
+        for (var x = 0; x < elements.length; x++) {
+            var links = elements[x].getElementsByTagName('A');
+            for (var y = 0; y < links.length; y++) {
+                let new_querystring = links[y].getAttribute('href');
+                links[y].onclick = function(e) {
+                    e.preventDefault();
+                    // Store the query string in sessiong storage.
+                    window.sessionStorage.setItem("job_results_current_page", new_querystring);
```

Comment:
Can we use something more generic here like `ajax_tables_current_page` or something? We could potentially use django-ajax-tables for other views besides job-results in the future.